### ISSUE TITLE
failing action, bioconda channel not available

### ISSUE THREAD

**Issue by violafanfani** [s-weigand/setup-conda#235]
**Describe the bug**
I use the action to test and release a python package with the gihub CI. Lest week all worked correctly, but I am now getting an error:
`UnavailableInvalidChannel: HTTP 404 NOT FOUND for channel  bioconda <https://conda.anaconda.org/ bioconda>` when creating the environment (`/usr/share/miniconda/bin/conda create -y -n __setup_conda python=3.9`). The bioconda channels doesn't seem to be down, as I tried generating the same environment on my local machine.

**To Reproduce**
1. Go to netzoopy actions : 'https://github.com/netZoo/netZooPy/runs/6628979901?check_suite_focus=true'
2. Click on 'actions'
3. Scroll down to `test conda #44`

**Expected behavior**
The rule is supposed to run with no errors.

**Additional context**

Rule in CI:
```
      - name: "Setup Conda"
        uses: s-weigand/setup-conda@v1
        with:
          update-conda: true
          python-version: ${{ matrix.python-version }}
          conda-channels: anaconda, conda-forge, bioconda
```

Entire error:
```
Installing conda python 3.9
  /usr/share/miniconda/bin/conda create -y -n __setup_conda python=3.9
  Collecting package metadata (current_repodata.json): ...working... failed
  
  UnavailableInvalidChannel: HTTP 404 NOT FOUND for channel  bioconda <https://conda.anaconda.org/ bioconda>
  
  The channel is not accessible or is invalid.
  
  You will need to adjust your conda configuration to proceed.
  Use `conda config --show channels` to view your configuration's current state,
  and use `conda config --show-sources` to view config file locations.
  
  
  Error: The process '/usr/share/miniconda/bin/conda' failed with exit code 1
```

**Comment by s-weigand** [comment]
Hi @violafanfani , thanks for filing this issue.
I found that the root of the problem, after splitting the channel at `,` I do not strip the wite spaces 🤦‍♀️
I'm working on a fix, but if you don't want to wait you can do this hot fix on your side
```diff
- conda-channels: anaconda, conda-forge, bioconda
+ conda-channels: anaconda,conda-forge,bioconda
```

**Comment by s-weigand** [comment]
@all-contributors please add @violafanfani  for bug

**Comment by violafanfani** [comment]
Thanks for the quick reply, I will do that to be on the safe side. In the meantime I also checked what could have been going wrong, it looks like the issue has started with conda 4.13 (it was working last week with conda 4.12). Maybe a different behavior?

**Comment by s-weigand** [comment]
@violafanfani Yeah that might be related to the conda version 🤔

The way conda is called is via `@actions/exec`'s `exec` function which takes a list for the arguments.
https://github.com/s-weigand/setup-conda/blob/b8347dabbcb6986ed9242a881d4d2628175bdcd0/src/conda_actions.ts#L220

Anyway, I thought that the ts `replace` behaves like the python one which is why only the first whitespace was replaced. And ofc my test case only had a single white space so this wasn't so it passed... 🤦‍♀️🙈 
So even if it worked with a different conda version this was a bug on the action side that would have broken with two leading whitespaces and the new conda behavior.
Now the unittest inputs are more robust
https://github.com/s-weigand/setup-conda/blob/b8347dabbcb6986ed9242a881d4d2628175bdcd0/__tests__/load_config.test.ts#L8

I just released version `v1.1.1` which also repointed the `v1` tag, so if you just rerun your failed workflow it should pass now.
It would be awesome if you could confirm this and close the issue if it was fixed.

**Comment by violafanfani** [comment]
Yes, I can confirm that the issue is now corrected. Thanks for your help!
